packages/common-substrate/src/project/versioned/ProjectManifestVersioned.ts

```diff
@@ -43,10 +43,6 @@ export class SubstrateProjectManifestVersioned implements ISubstrateProjectManif
     return this._impl as ProjectManifestV1_0_0Impl;
   }
 
-  toDeployment(): string | undefined {
```

Comment:
I think its handy to retain this function or a `get deployment()` function to avoid doing x.asImpl.deployment